pkg/blockfactory/blockissuer.go

```diff
@@ -67,6 +68,28 @@ func (i *BlockIssuer) Shutdown() {
 func (i *BlockIssuer) CreateValidationBlock(ctx context.Context, issuerAccount Account, opts ...options.Option[ValidatorBlockParams]) (*model.Block, error) {
 	blockParams := options.Apply(&ValidatorBlockParams{}, opts)
 
+	if blockParams.BlockHeader.IssuingTime == nil {
+		issuingTime := time.Now().UTC()
+		blockParams.BlockHeader.IssuingTime = &issuingTime
+	}
+
+	if blockParams.BlockHeader.SlotCommitment == nil {
+		var err error
+		blockParams.BlockHeader.SlotCommitment, err = i.getCommitment(i.protocol.CurrentAPI().TimeProvider().SlotFromTime(*blockParams.BlockHeader.IssuingTime))
```

Comment:
The name of the `getCommitment` method is misleading: it seems you are getting the commitment corresponding to the issuingtime. Maybe `getAddressableCommittment` or something like that?